vectordatasource/meta/sql.py

```diff
@@ -10,6 +10,7 @@
         'ne_10m_admin_0_boundary_lines_land',
         'ne_10m_admin_0_boundary_lines_map_units',
         'ne_10m_admin_0_boundary_lines_disputed_areas',
+        'ne_10m_admin_0_boundary_lines_disputed_areas_chn',
```

Comment:
@nvkelso you didn't include this line of change in the issue, but I think it is necessary too?

Reply:
Got catch! I agree it's neccesary.